### PR TITLE
fix(scripts): improve inactive account uids script

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - there were a few small issues with the inactive account uids script

This commit:
 - stops opening an output file in dry run mode
 - fixes an import so that the tsc'd js won't error
 - eliminates the accounts with Stripe subscriptions in the initial query

### CODE REVIEW

File: packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts [hunk 1]
```diff
@@ -161,19 +159,25 @@ const init = async () => {
     ])
     .as('securityEventUids');
 
+  const accountCustomerUids = AccountCustomers.query()
+    .distinct('uid')
```
Comment (dschom) [inline packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts:163]: Are there non-distinct uids?

Comment (chenba) [reply packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts:163]: No, I just got copy-paste happy.  Thank you for catching that.

File: packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts [hunk 2]
```diff
@@ -231,6 +230,8 @@ const init = async () => {
     return 0;
   }
 
+  const fd = fs.openSync(filepath, 'a');
```
Comment (dschom) [inline packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts:233]: `node:fs/promises` supports async/await for file io, not that it really matters here.

**Review by dschom — APPROVED** [review]
LGTM

**Pull request merged**